dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/tracker/Assertions.java

```diff
@@ -293,6 +303,23 @@ public static void assertSlimPager(int pageNumber, int pageSize, boolean isLast,
                 isLast ? "should be the last page" : "should NOT be the last page"));
   }
 
+  public static void assertHasTimeStamp(Date date) {
+    assertTrue(
+        hasTimeStamp(date),
+        String.format("Supported format is %s but found %s", DATE_WITH_TIMESTAMP_PATTERN, date));
+  }
+
+  private static boolean hasTimeStamp(Date date) {
+    try {
+
```

Comment:
nit: remove this newline

Reply:
shouldnt it be taken care by our formatter ?

Reply:
it seems to be ok with some extra newlines 😁